RecoTracker/MkFitCore/src/CandCloner.cc

```diff
@@ -191,10 +191,12 @@ namespace mkfit {
           if (n_pushed >= mp_iteration_params->maxCandsPerSeed)
             break;
 
-          // set the overlap if we have a true hit and pT > pTCutOverlap
+          // set the overlap if we have a true hit (and possibly pT > pTCutOverlap)
           HitMatch *hm;
-          if (tc.pT() > mp_iteration_params->pTCutOverlap && h2a.hitIdx >= 0 &&
-              (hm = ccand[h2a.trkIdx].findOverlap(h2a.hitIdx, h2a.module))) {
+          if (tc.pT() > 0.0 && h2a.hitIdx >= 0 &&
+              (hm = ccand[h2a.trkIdx].findOverlap(
+                   h2a.hitIdx,
+                   h2a.module))) {  // To select tracks with a different pT cut, can use: mp_iteration_params->pTCutOverlap
```

Comment:
please move the comment out of line with code to avoid line splits in the code).

Why not simply change the cut value to 0 in the configuration?

Reply:
Changing the value in the configuration would require remaking JSON files, thus I thought we could avoid it.

> please move the comment out of line with code to avoid line splits in the code).
OK.